core/sources/domains.move

```diff
@@ -438,6 +493,39 @@ module aptos_names::domains {
         };
     }
 
+    public fun set_reverse_lookup(account: &signer, key: &NameRecordKeyV1) acquires NameRegistryV1, ReverseLookupRegistryV1, SetNameAddressEventsV1, SetReverseLookupEventsV1 {
```

Comment:
Can you add documentation for this public function?

Reply:
Done